### DISCUSSION TITLE
Evennia Mudlet Mapper

### DISCUSSION THREAD

**Discussion by RodRitter** [evennia/evennia#2718] [Community Contribs & Snippets]
# Evennia Mudlet Mapper
I've recently implemented some auto-mapping for Mudlet's built-in mapper. Hopefully, it'll be helpful to others as well. Please excuse the bad python code. I'm not a python dev.

[Repository with Setup & Instructions](https://github.com/RodRitter/Evennia-Doodads/tree/main/Evennia%20Mudlet%20Mapper) 

The idea is to use a Mapper character, who can run around and make the map. The map can be exported and shared.

![image](https://user-images.githubusercontent.com/4159679/162093437-7a88069e-17a7-4f0d-96ee-48a3dea8cc5b.png)

**Comment by Griatch** [comment]
Very cool! Thanks for making an easy-to-use example. 
Also a good example of using GMCP from Evennia (as simple as, in this case, `obj.msg(location=dict_of_data)`). :grin:
